idd/Energy+.idd.in

```diff
@@ -107248,117 +107332,122 @@ OutputControl:Files,
        \key Yes
        \key No
        \default Yes
-  A9 , \field Output Zone Sizing
+  A9 , \field Output Space Sizing
```

Comment:
New output file for space sizing spsz, equivalent to zsz output. This new field inserted in OutputControl:Files triggers a transition rule.